environments/py-3.10-linux-64-dev.conda.lock.yml

```diff
@@ -163,7 +163,7 @@ dependencies:
   - mdurl=0.1.2=pyhd8ed1ab_1
   - metis=5.1.0=hd0bcaf9_1007
   - mistune=3.1.2=pyhd8ed1ab_0
-  - mkl=2024.2.2=ha957f24_16
+  - mkl=2023.2.0=h84fe81f_50496
```

Comment:
@domfournier, no, it cannot be back to 2023, as it not compatible with the version of pytorch we use in other packages

Reply:
Windows tests don't pass otherwise